### ISSUE TITLE
autodoc renders weird result for default parameter []

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1519]
.. autoclass:: m.M

```
class M(object):
    def __init__(self, m=[], a=[], b=[], c=[]):
        """Init for M"""
        pass
```

And I got::

```
class m.M(m=, []a=, []b=, []c=[])
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1519
- Originally reported by: [fengsp](https://bitbucket.org/fengsp)
- Originally created at: 2014-07-25T06:13:27.971

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-07-25 21:00:57+00:00_

Duplicate of #1503.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-07-25 21:01:15+00:00_

The issue you reported will be fixed at Sphinx-1.2.3. Thank you for your reporting!
